app/src/main/java/org/mozilla/tv/firefox/webrender/FocusedDOMElementCacheInterface.kt

```diff
@@ -19,7 +19,10 @@ import android.view.View
  *
  * It is important that this value be cached just before the WebView regains focus.  Otherwise
  * edge cases present themselves where a DOM element is cached, a new element is focused by a
- * site, and then we return focus to the earlier (now incorrect) element (see #1830).
+ * site, and then we return focus to the earlier (now incorrect) element (see #1830).  This
```

Comment:
Nit: Newline break before "This"

Reply:
Are you sure?  It seems to fit with that paragraph, and it's shorter than other lines.